api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/GraphProcess.scala

```diff
@@ -2,17 +2,18 @@ package pl.touk.nussknacker.engine.api.deployment
 
 import io.circe.parser.parse
 
-//object GraphProcess {
-//
-//  def apply(jsonString: String): GraphProcess = {
-//    val json = parse(jsonString) match {
-//      case Left(_) => throw new IllegalArgumentException(s"Invalid raw json string: $jsonString.")
-//      case Right(json) => json.spaces2
-//    }
-//
-//    new GraphProcess(json)
-//  }
-//
-//}
+object GraphProcess {
 
-final case class GraphProcess(processAsJson: String) extends AnyVal
+  //TODO: Add json validation here?
+  def apply(jsonString: String): GraphProcess = {
+    val json = parse(jsonString) match {
+      case Left(_) => throw new IllegalArgumentException(s"Invalid raw json string: $jsonString.")
+      case Right(json) => json.spaces2
```

Comment:
if we parse it, let's have Json inside... :)

Reply:
yep, I know it. Right now I've left it as string, because it was easier.